### PR TITLE
Propose talk for deepstream.io

### PR DESCRIPTION
**Pull request by yasserf**
Not sure what the maximum amount of talks are ( seems to be three ) but wanted to raise as a placeholder until a slot is available.

Thanks!

### CODE REVIEW

**Comment by rmehner** [comment]
Hi there,

I missed to update the site for the next meetup. There won't be a meetup in September, because we run [Reject.JS](http://rejectjs.org) then, the next meetup would be on October 15th. Does this still work for you?

**Comment by yasserf** [comment]
Yep, I figured as much. That's totally fine.

File: index.html [hunk 1]
```diff
@@ -112,20 +111,6 @@
                 Twitter</a> or <a href="https://github.com/berlinjs/berlinjs.org">create a pull request on GitHub</a> to this page right away.
               </p>
             </div>
-
-            <!--
```
Comment (yasserf) [inline index.html:116]: This appears to be a duplicate of line 66:
    <!-- THIS IS WHAT A TALK ABSTRACT LOOKS LIKE

**Comment by rmehner** [comment]
Rebased everything, here it is: 2ca9778ac394d16b0c1a231d433b5411e72ec7c9

Looking forward to meeting you in October!

**Pull request closed without merging**